src/components/body/footer/Footer.tsx

```diff
@@ -1,4 +1,55 @@
-const Footer = () => {
-  return <div className="footer">footer</div>;
+import {Link} from "react-router-dom";
+import LANGUAGES from '../../translations/constants';
+import {FormattedMessage } from 'react-intl';
+
+type FooterProps = {
+  onLanguageSwitch: (arg: string) => void
+}
+const Footer = (props: FooterProps) => {
+  const { onLanguageSwitch } = props
+
+  return (
+    <div style={{display:'flex'}}>
+      <Link to="/">
+        <div>
+          <FormattedMessage id="app.name"/>
+        </div>
+      </Link>
+      <Link to="/categories">
+        <div>
+          <FormattedMessage id="app.categories"/>
```

Comment:
Pleasewrite consistently:

with space before closing of tag symbol or with it:

 id="app.categories" /&gt;
 
 or
 
  id="app.categories"/&gt;
  
  You can setup a plugin to fix such errors